src/commands/set.js

```diff
@@ -68,39 +81,47 @@ const attemptWriteToFile = (variable, value) => {
 	return result;
 };
 
-const setBoolean = (variable, path) => value => {
+const setBoolean = (dotNotation, value) => {
 	if (!checkBoolean(value)) {
 		throw new ValidationError('Value must be a boolean.');
 	}
-
+	const dotNotationArray = dotNotation.split('.');
 	const newValue = value === 'true';
-	path.reduce(setNestedConfigProperty(newValue), config);
+	dotNotationArray.reduce(setNestedConfigProperty(newValue), config);
 
-	if (variable === 'testnet') {
+	if (dotNotation === 'api.testnet') {
 		liskAPIInstance.setTestnet(newValue);
 	}
 
-	return attemptWriteToFile(variable, value);
+	if (dotNotation === 'api.ssl') {
+		liskAPIInstance.setSSL(newValue);
+	}
+
+	return attemptWriteToFile(value, dotNotationArray);
```

Comment:
Why not pass `dotNotation` here (and in `setString`)? Then `attemptWriteToFile` wouldn't have to reconstruct it. (Sorry, I know this is the opposite of my previous review, I only just realised.)

Reply:
You mean to stringify it for the message? I do not think that there is lot of processing power involved.

Reply:
It's not about processing power, it's about readability. Fewer processing steps = easier to read and understand.

Reply:
Agree with @willclarktech 
In this case, the `dotNotation` is not used as Array, but just for displaying as it is, so I think it should be passed as `string`